distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -457,6 +457,8 @@ generate_query dialect query = case query of
     Query.Drop_Table name if_exists ->
         maybe_if_exists = if if_exists then Builder.code "IF EXISTS " else Builder.empty
         Builder.code "DROP TABLE " ++ maybe_if_exists ++ dialect.wrap_identifier name
+    Query.Truncate_Table name ->
+        Builder.code "DELETE FROM " ++ dialect.wrap_identifier name
```

Comment:
Why not `TRUNCATE TABLE ...`?

Reply:
Not available in SQLite. I guess I could try to do it in Postgres.